hs_core/hydroshare/utils.py

```diff
@@ -273,8 +426,9 @@ def validate_resource_file_count(resource_cls, files, resource=None):
                 raise ResourceFileValidationException(err_msg)
 
 
-def resource_pre_create_actions(resource_type, resource_title, page_redirect_url_key, files=(), metadata=None,
-                                **kwargs):
+def resource_pre_create_actions(resource_type, resource_title, page_redirect_url_key,
```

Comment:
This function is being used by one of the REST endpoint. Though the changes to this function not likely to break the REST API, the REST API needs to change to support user IRODS zone. Should create an issue so that it can be addressed later.